Fluent should not crash authfe when unavailable

For example, when fluent is not running authfe fails with:

```
Dear computer, kubectl logs authfe-3803633268-ufrif authfe
INFO: 2016/08/31 10:22:28.383735 Listening on :80
panic: fluent#reconnect: failed to reconnect!

goroutine 13 [running]:
panic(0xa54240, 0xc8201b72f0)
        /usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/weaveworks/service/vendor/github.com/fluent/fluent-logger-golang/fluent.(*Fluent).reconnect.func1(0xc820122210)
        /go/src/github.com/weaveworks/service/vendor/github.com/fluent/fluent-logger-golang/fluent/fluent.go:261 +0x102
created by github.com/weaveworks/service/vendor/github.com/fluent/fluent-logger-golang/fluent.(*Fluent).reconnect
        /go/src/github.com/weaveworks/service/vendor/github.com/fluent/fluent-logger-golang/fluent/fluent.go:267 +0x35
```

--- a/common/logging/event.go
+++ b/common/logging/event.go
@@ -45,6 +45,7 @@ func NewEventLogger(fluentHostPort string) (*EventLogger, error) {
 		FluentPort:   intPort,
 		FluentHost:   host,
 		AsyncConnect: true,
+		MaxRetry:     -1,
 	})
 	if err != nil {
 		return nil, err

